transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,31 @@
+{{
+config({
+ "tags":"hourly",
+})
+}}
+
+{%- set include_columns = ["context_page_referrer", "channel", "context_app_namespace", "user_actual_id"
```

Comment:
Do we need all these columns? For example `uuid_ts` is used for debugging purposes (according to [Rudderstack's doc](https://www.rudderstack.com/docs/destinations/warehouse-destinations/warehouse-schema/)).

`context_*` fields is another category. For example `context_library_name` is the name of the rudderstack library.

Reply:
I copied the columns that are in Looker, I'll check with @hmhealey tomorrow to see what fields they use from performance events, and we can probably drop most of the fields.

Reply:
https://mattermost.atlassian.net/browse/MM-50797

Reply:
@sri-byte, are we sure that MM-50797 is the cleanup ticket Yiannis was referring to?

Reply:
We probably don't need these, The next step for me is to clean the LookML view code, and then remove these fields from here.